cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -170,18 +171,21 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
         val config = globalOptionsForSubcommands.config
         val analyzer = Analyzer(config.analyzer)
 
+        val repositoryConfiguration = actualRepositoryConfigurationFile?.readValueOrNull() ?: RepositoryConfiguration()
+
         val curationProvider = FallbackPackageCurationProvider(
             listOfNotNull(
                 FilePackageCurationProvider.from(packageCurationsFile, packageCurationsDir),
                 config.analyzer.sw360Configuration?.let {
                     Sw360PackageCurationProvider(it).takeIf { useSw360Curations }
                 },
+                SimplePackageCurationProvider(repositoryConfiguration.curations.packages).takeIf {
+                    repositoryConfiguration.curations.packages.isNotEmpty()
```

Comment:
This fallback provider is taking only the curations from the first provider in the list which has at least one.

I believe that the first item in the list should be a provider which return the union of local and global package curations.
Both, global and local configuration should always apply. BTW I believe that having mulitple curations per package is perfectly fine, we should not assume that the global configuration only contains one curation per package, nor should we assume the same for the local configuration and also not for their union.

Reply:
Done.
Please also see the added description in the commit message.

Regarding the order. What should be the curation that takes precedence if it's conflicting with another one?
With the implemented change it is local > global because this is how it was discussed for a similar discussion with `license choices` with @mnonnenmacher.

Reply:
I agree with the order.

Reply:
Thanks!